.eslintrc.json

```diff
@@ -1,12 +1,8 @@
 {
-  "env": {
-    "node": true
```

Comment:
Don’t we want this for `webpack.config.js`?

Reply:
Added to `webpack.config.js`